From d4df006e4a1680ff44eae4d1b447295340a9b376 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 7 Apr 2019 18:55:22 +0300
Subject: [PATCH 13/13] cl-som-imx8: defconfig: Update defconfig

Enable the options that were removed by cleanup by mistake.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/configs/cl-som-imx8_defconfig | 27 +++++++++++----------------
 1 file changed, 11 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/configs/cl-som-imx8_defconfig b/arch/arm64/configs/cl-som-imx8_defconfig
index 3a247b3..285f195 100644
--- a/arch/arm64/configs/cl-som-imx8_defconfig
+++ b/arch/arm64/configs/cl-som-imx8_defconfig
@@ -13,6 +13,7 @@ CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=14
+CONFIG_NUMA_BALANCING=y
 CONFIG_MEMCG=y
 CONFIG_MEMCG_SWAP=y
 CONFIG_BLK_CGROUP=y
@@ -25,12 +26,12 @@ CONFIG_CGROUP_PERF=y
 CONFIG_USER_NS=y
 CONFIG_SCHED_AUTOGROUP=y
 CONFIG_BLK_DEV_INITRD=y
-CONFIG_KALLSYMS_ALL=y
 CONFIG_PROFILING=y
 CONFIG_JUMP_LABEL=y
 CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 CONFIG_ARCH_FSL_IMX8QM=y
+CONFIG_ARCH_FSL_IMX8QP=y
 CONFIG_ARCH_FSL_IMX8QXP=y
 CONFIG_ARCH_FSL_IMX8MQ=y
 CONFIG_ARCH_FSL_IMX8MM=y
@@ -42,6 +43,7 @@ CONFIG_PCI_HOST_GENERIC=y
 CONFIG_PCI_XGENE=y
 CONFIG_ARM64_VA_BITS_48=y
 CONFIG_SCHED_MC=y
+CONFIG_NUMA=y
 CONFIG_PREEMPT=y
 CONFIG_KSM=y
 CONFIG_TRANSPARENT_HUGEPAGE=y
@@ -49,7 +51,6 @@ CONFIG_CMA=y
 CONFIG_SECCOMP=y
 CONFIG_KEXEC=y
 CONFIG_XEN=y
-# CONFIG_EFI is not set
 CONFIG_COMPAT=y
 CONFIG_PM_DEBUG=y
 CONFIG_PM_TEST_SUSPEND=y
@@ -122,6 +123,7 @@ CONFIG_BT_HCIUART_INTEL=y
 CONFIG_BT_HCIUART_QCA=y
 CONFIG_BT_HCIVHCI=y
 CONFIG_CFG80211=y
+CONFIG_CFG80211_WEXT=y
 CONFIG_MAC80211=m
 CONFIG_RFKILL=y
 CONFIG_NET_9P=y
@@ -131,7 +133,8 @@ CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_DMA_CMA=y
 CONFIG_CMA_SIZE_MBYTES=320
-CONFIG_VEXPRESS_CONFIG=y
+CONFIG_ARM_CCI400_PMU=y
+CONFIG_ARM_CCI5xx_PMU=y
 CONFIG_MTD=y
 CONFIG_MTD_CMDLINE_PARTS=y
 CONFIG_MTD_BLOCK=y
@@ -143,6 +146,7 @@ CONFIG_MTD_NAND_GPMI_NAND=y
 CONFIG_MTD_SPI_NOR=y
 CONFIG_SPI_FSL_FLEXSPI=y
 CONFIG_MTD_UBI=y
+CONFIG_OF_OVERLAY=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_NBD=m
 CONFIG_XEN_BLKDEV_BACKEND=m
@@ -188,8 +192,6 @@ CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
 CONFIG_USB_USBNET=y
-CONFIG_USB_NET_CDCETHER=m
-CONFIG_USB_NET_CDC_NCM=m
 CONFIG_USB_NET_DM9601=m
 CONFIG_USB_NET_SR9800=m
 CONFIG_USB_NET_SMSC75XX=m
@@ -197,7 +199,6 @@ CONFIG_USB_NET_SMSC95XX=m
 CONFIG_USB_NET_NET1080=m
 CONFIG_USB_NET_PLUSB=m
 CONFIG_USB_NET_MCS7830=y
-CONFIG_USB_NET_CDC_SUBSET=m
 CONFIG_USB_NET_ZAURUS=m
 CONFIG_IWLWIFI=m
 CONFIG_IWLDVM=m
@@ -245,9 +246,9 @@ CONFIG_GPIO_XGENE=y
 CONFIG_GPIO_PCA953X=y
 CONFIG_GPIO_PCA953X_IRQ=y
 CONFIG_GPIO_MAX77620=y
-CONFIG_POWER_RESET_VEXPRESS=y
 CONFIG_POWER_RESET_XGENE=y
 CONFIG_POWER_RESET_SYSCON=y
+CONFIG_SYSCON_REBOOT_MODE=y
 CONFIG_BATTERY_BQ27XXX=y
 CONFIG_SENSORS_ARM_SCPI=y
 CONFIG_SENSORS_LM90=m
@@ -290,10 +291,7 @@ CONFIG_MXC_CAMERA_OV5640_MIPI_V2=y
 CONFIG_V4L_MEM2MEM_DRIVERS=y
 CONFIG_IMX_DPU_CORE=y
 CONFIG_IMX_DCSS_CORE=y
-CONFIG_IMX_LCDIF_CORE=y
 CONFIG_DRM=y
-CONFIG_DRM_I2C_CH7006=m
-CONFIG_DRM_I2C_SIL164=m
 CONFIG_DRM_VIVANTE=y
 CONFIG_DRM_PANEL_SIMPLE=m
 CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
@@ -421,25 +419,22 @@ CONFIG_VIRTIO_MMIO=y
 CONFIG_XEN_GNTDEV=y
 CONFIG_XEN_GRANT_DEV_ALLOC=y
 CONFIG_STAGING=y
-CONFIG_STAGING_MEDIA=y
 CONFIG_ION=y
 CONFIG_ION_SYSTEM_HEAP=y
 CONFIG_ION_CMA_HEAP=y
 CONFIG_TYPEC_TCPM=y
 CONFIG_TYPEC_TCPCI=y
-CONFIG_COMMON_CLK_VERSATILE=y
-CONFIG_CLK_SP810=y
-CONFIG_CLK_VEXPRESS_OSC=y
 CONFIG_COMMON_CLK_SCPI=y
 CONFIG_COMMON_CLK_CS2000_CP=y
 CONFIG_COMMON_CLK_S2MPS11=y
 CONFIG_CLK_QORIQ=y
-CONFIG_ARM_TIMER_SP804=y
 CONFIG_CLKSRC_IMX_SYS_CNT=y
 CONFIG_MAILBOX=y
 CONFIG_ARM_MHU=y
+CONFIG_PLATFORM_MHU=y
 CONFIG_BCM_FLEXRM_MBOX=y
 CONFIG_ARM_SMMU=y
+CONFIG_ARM_SMMU_V3=y
 CONFIG_ARCH_MXC_ARM64=y
 CONFIG_EXTCON_PTN5150=y
 CONFIG_IIO=y
@@ -482,8 +477,8 @@ CONFIG_9P_FS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_PRINTK_TIME=y
+CONFIG_DEBUG_FS=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_KERNEL=y
 CONFIG_MEMTEST=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
-- 
1.9.1

